src/cmd/troubleshoot/image_test.go

```diff
@@ -1,16 +1,17 @@
 package troubleshoot
```

Comment:
this amount of tests feel a bit overkill 😅

Reply:
I didn't add any, just compressed them a wee bit.